donald_wade/week_01/001/js-homework-1.js

```diff
@@ -6,25 +6,25 @@ function squareNumber(num) {
   console.log(typeof(Number(num)));
   newNum = Number(num);
   result = newNum * newNum;
-  document.write("Your number (" + newNum + ") squared is " + result);
+  document.getElementById('square').innerHTML += "<p>Your new number (" + newNum + ") squared is " + result + "</p>";
 }
 
 function halfNumber(num) {
   var num = document.getElementsByName('num2')[0].value;
```

Comment:
that is a part I wanted to make an accent to. You cannot reassign parameter inside of the function, if you do so in that case you simply don't need a parameter.